src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let res = days * 40;
```

Comment:
What mean `res`? Maybe totalCost?

Reply:
Also add var for a magic numbers like 40, 50...

Reply:
I had added these changes 5 seconds before you wrote this, my commit is above this message)

Reply:
Maybe you didn't see it